react/getting_started_with_react/rendering_techniques.md

```diff
@@ -99,24 +99,12 @@ function App() {
     </div>
   );
 }
-~~~
+```
 
 <div class="lesson-note lesson-note--tip" markdown="1">
 #### "Missing in props validation"
-  You may notice squiggly lines under your props (for example under `color` and
-  `fontSize` inside the Button component below). Hovering over these will tell
-  you they are `missing in props validation`. For now, this can safely be
-  ignored as it is just a default ESLint rule warning about prop types,
-  something that will be covered later in the course.
-
-  For now, you may want to turn off this rule by adding the following to your `.eslintrc.cjs` file:
-  
-  ```javascript
-  "rules": {
-    // Your other rules
-    "react/prop-types": "off"
-  }
-  ```
+
+You may notice squiggly lines under your props (for example under `color` and `fontSize` inside the Button component below). Hovering over these will tell you they are `missing in props validation`. For now, this can safely be ignored as it is just a default ESLint rule warning about prop types, something that will be covered later in the course.
```

Comment:
````suggestion
You may notice squiggly lines under your props (for example under `color` and `fontSize` inside the Button component below). Hovering over these will tell you they are `missing in props validation`. For now, this can safely be ignored as it is just a default ESLint rule warning about prop types, something that will be covered later in the course.

For now, you may want to turn off this rule by adding the following to your `.eslintrc.cjs` file:

```javascript
"rules": {
  // Your other rules
  "react/prop-types": "off"
}
```
````

Just this last thing to fix - you accidentally got rid of one of the parts we needed to keep in the merge conflict (no worries - I probably wasn't as clear as I could've been for your first conflict resolution!). If you can add this back as shown above, all should be good to merge after!